util.hpp

```diff
@@ -314,4 +314,14 @@ vector<string> getFileHeaders(const string& header, const string& filename) {
     return headers;
 }
 
+// converts number to a string form, but with padding
+// does nothing if stringLength is longer than digits in number

```

Comment:
// converts number to a string padded with leading zeros
// does nothing if the number of digits is greater equals stringLength

Reply:
thanks a lot! this is what happens when I write things at 11pm :O